### PR TITLE
Tooltip support for annotations and series data refresh fix

### PR DESCRIPTION
**Pull request by mrtncls**
### Checklist

- [x] I have included examples or tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file
- [ ] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- Adds support for WPF tooltips on Annotations
- Fixes refresh series when not visible (WPF)

@oxyplot/admins

### CODE REVIEW

File: Source/OxyPlot.Wpf/Annotations/Annotation.cs [hunk 1]
```diff
@@ -113,6 +113,7 @@ public virtual void SynchronizeProperties()
             a.Layer = this.Layer;
             a.XAxisKey = this.XAxisKey;
             a.YAxisKey = this.YAxisKey;
+            a.ToolTip = this.ToolTip as string;
```
Comment (objorke) [inline Source/OxyPlot.Wpf/Annotations/Annotation.cs]: OK. Thanks!

File: Source/OxyPlot.Wpf/Series/Series.cs [hunk 2]
```diff
@@ -64,7 +64,7 @@ public abstract class Series : ItemsControl
         /// </summary>
         static Series()
         {
-            VisibilityProperty.OverrideMetadata(typeof(Series), new PropertyMetadata(Visibility.Visible, AppearanceChanged));
+            VisibilityProperty.OverrideMetadata(typeof(Series), new PropertyMetadata(Visibility.Visible, DataChanged));
```
Comment (objorke) [inline Source/OxyPlot.Wpf/Series/Series.cs]: It would be great to have an example showing this issue!

**Comment by objorke** [comment]
Please update the change log.

File: Source/OxyPlot.Wpf/Series/Series.cs (hunk 2 above)
Comment (mrtncls) [inline Source/OxyPlot.Wpf/Series/Series.cs]: Small example added. See Examples/WPF/LiveDemo

Steps to reproduces:
1. Enable both checkboxes (show and generate) for a few seconds. You should see data plotting.
2. Uncheck both
3. Toggle 'show' and verfiy data is still there
4. Check 'generate' and wait a few seconds before unchecking
5. Toggle 'show' to check if data is correct.

Prior the fix, the old data would still be shown.

Hope all it's clear.

**Comment by objorke** [comment]
I created #969 and #970

**Comment by objorke** [comment]
See #971. I added examples and moved the example commits before the fixes. That makes it easier to check that the bug is actually fixed.

**Comment by objorke** [comment]
I have merged #971.
Thanks for contributing!!

**Pull request closed without merging**